hack/helm/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v2
+name: hnc
+description: A Helm chart to install hnc on Kubernetes
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.0
```

Comment:
I'm not clear on what this is supposed to be. E.g. if we update HNC from v0.8.0 to v0.9.0, should this go from v0.1.0 to v0.2.0?

Reply:
my understanding which could be incorrect:

- chart version changes when the helm chart itself changes
- app version changes when the application itself changes

so appVersion should be part of the release process to auto bump it. I guess chart version bumps handled in the PR template as process(?)